### PR TITLE
[Integrate] As a user, I need to grant PERMISSION to allow the app to read the incoming messages

### LINKED ISSUE #23: [Integrate] As a user, I need to grant PERMISSION to allow the app to read the incoming messages
## Why

User need to grant permission to allow the app to read the incoming messages

## Acceptance Criteria

- Request runtime permission once user navigates to home screen:
    - `Manifest.permission.RECEIVE_SMS`
- If permission is not granted, request user to grant the permission
- If permission is denied, show toast message "Permission denied"
- If permission is denied and never ask again, show toast message "Permission never ask again"

### PR DESCRIPTION
**Pull request by hoangnguyen92dn**
## What happened 👀

User need to grant permission to allow the app to read the incoming messages
 
## Insight 📝

- Request `RECEIVE_SMS` permission on Home screen
 
## Proof Of Work 📹

Show us the proof that it worked (photo, gif...etc)

### CODE REVIEW

**Review by lydiasama — DISMISSED** [review]

**Review by luongvo — APPROVED** [review]

File: android/app/src/main/java/co/nimblehq/smsforwarder/ui/screens/home/HomeFragment.kt [hunk 1]
```diff
@@ -65,6 +71,32 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>() {
...
+                // Granted
+                toaster.display("Permission granted")
+            }
+            permission.shouldShowRequestPermissionRationale -> {
+                // Deny
+                toaster.display("Permission denied")
+            }
+            else -> {
+                // Deny and never ask again
+                toaster.display("Permission never ask again")
```
Comment (lydiasama) [inline android/app/src/main/java/co/nimblehq/smsforwarder/ui/screens/home/HomeFragment.kt]: Should we extract these hard-coded strings to the string resource?

**Comment by lydiasama** [comment]
@hoangnguyen92dn Could you please rebase on develop branch so that #7  can see the only change commit? 🙏🏻

**Comment by hoangnguyen92dn** [comment]
@lydiasama Do you have new comments on this PR? 🙏

**Comment by lydiasama** [comment]
@hoangnguyen92dn That's all. Apart from https://github.com/nimblehq/sms-forwarder/pull/6#discussion_r639892982 is LGTM.

**Comment by hoangnguyen92dn** [comment]
@lydiasama So what is this? 🤣 
> Change my mind about the hard-coded strings

**Comment by lydiasama** [comment]
@hoangnguyen92dn It's the dismissing my review from approved. 😂

**Review by lydiasama — APPROVED** [review]

**Pull request merged**